pom.xml

```diff
@@ -153,32 +166,7 @@
             <dependency>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
-                <version>2.8.0</version>
-            </dependency>
-            <dependency>
```

Comment:
no idea why any of this was being pulled in, everything still worked after removing the below dependencies